src/main/resources/rs117/hd/uniforms/water_types.glsl

```diff
@@ -13,11 +13,8 @@ struct WaterType
     float duration;
     float fresnelAmount;
     vec3 surfaceColor;
-    float pad0;
     vec3 foamColor;
-    float pad1;
     vec3 depthColor;
-    float pad2;
```

Comment:
If I remember correctly, removing this padding is slightly dangerous, but I guess since we're mostly supporting OpenGL 4.3 and up, it should be fine. This will need testing on MacOS though for sure.

Reply:
I reckon if we find that we still need explicit padding, we can leave it in for now, and generate it when we get to generating the GLSL for these UBOs.